tools/cmake/idf.cmake

```diff
@@ -16,6 +16,8 @@ if(NOT __idf_env_set)
 
         if(NOT idf_path STREQUAL _idf_path)
             message(WARNING "IDF_PATH environment variable is different from inferred IDF_PATH.
+                            IDF_PATH environment variable: ${idf_path}
```

Comment:
This extra logging would not be needed in Zephyr as it automatically sets IDF_PATH. Did you have any issues prior this?

Reply:
I had an issue when setting up the environment and added this log to solve it.
I thought it could help others in the same situation. I had two idf folders and my environment was not coherent.

Reply:
I removed this commit